src/models/APISupport.ts

```diff
@@ -313,10 +318,12 @@ export class APISupport {
                 throw new Error("Error in running the test");
             }
             let startTime = new Date();
-            let portalUrl = testRunDao.portalUrl;
             let status = testRunDao.status;
             if(status == "ACCEPTED") {
-                console.log("View the load test run in progress at: "+ portalUrl)
+                console.log("\nView the load test run in Azure portal by following the steps:")
+                console.log("1. Go to your Azure Load Testing resource '"+this.resourceName+"' in subscription '"+this.subName+"'")
+                console.log("2. On the Tests page, go to test '"+this.testId+"'")
```

Comment:
This should be test display name?

Reply:
this was by mistake done, i changed in pr4.,